README.md

```diff
@@ -1 +1,195 @@
-# budget-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h1><b>SmartPocket Budget App</b></h1>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+## 📗 Table of Contents
+
+- [📖 Budget-App ](#-budget-app)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 SmartPocket Budget <a name="about-project"></a>
+
+A budget app named **SmartPocket Budget** keeps track of all your expenses associated with categories. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en/docs/Web/JavaScript">JavaScript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+The main features include:
+
+- Create new transactions to track your expenses.
+- Update or delete existing transactions.
+- Create new categories to classify your expenses by category.
+- View the calculated total amount of expenses per category.
+- Update or delete existing categories.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- The Live Demo is available [here](https://budget-app-trxc.onrender.com)
+- Video of the app is available [here](https://www.loom.com/share/25cd395985e94de5aae700c872a7fa5d)
```

Comment:
- [ ] Nice presentation, but you failed to highlight a piece of code in your editor that you found interesting or an app feature you felt proud of making.